ls1intum/Artemis
# [Feature] Open student submission in online editor

### Checklist

### Motivation and Context


Make it possible for tutors and instructors to review the code directly into Artemis instead of downloading the repo or checking it out via bitbucket.

### Description

- Use assessment-layout component for labouring
- Add code-editor container with individual nabber
- Reuse layout and functionality of dialog/modal component (which is now removed)

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create programming exercise and participate in programming exercise (also exam mode)
4. Assess programming exercise (via tutorDashboard, exercise-score, examTutorDashboard)
4.1 Try out all buttons and check if something strange happens
5. Complain about assessment
6. Assess complaint
6.1 Try out all buttons and check if something strange happens


### Screenshots


New layout for the assessment of programming exercises:
<img width="1417" alt="image" src="https://user-images.githubusercontent.com/41108487/91177520-22dc0d00-e6e4-11ea-999a-a5ab9d553ef5.png">

<img width="1377" alt="image" src="https://user-images.githubusercontent.com/41108487/91085436-bdd5d800-e64d-11ea-8a1c-34a0a35e846c.png">

Navbar elements are now below except automatic result (see screenshot 1)
<img width="1414" alt="image" src="https://user-images.githubusercontent.com/41108487/91086525-51f46f00-e64f-11ea-88ae-88f93c1df0f9.png">

Complaints:
Navbar elements are now below except automatic result (see screenshot 1)
![image](https://user-images.githubusercontent.com/41108487/91086863-ccbd8a00-e64f-11ea-8cf2-03806ae3480e.png)

![image](https://user-images.githubusercontent.com/41108487/91086883-d34c0180-e64f-11ea-8937-03f36bfd11dd.png)
